content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md

```diff
@@ -0,0 +1,138 @@
+
+### Removal of EphemeralContainers feature gate
+
+[Emphemeral containers](https://github.com/kubernetes/kubernetes/pull/111402) graduated to GA in v1.25. These are containers with a temporary duration that executes within namespaces of an existing pod. Ephemeral Containers are initiated by a user and intended to observe the state of other pods and containers for troubleshooting and debugging purposes. Hence the `EphemeralContainers` feature gate is always enabled and will be removed from the `--feature-gates` flag on the kube-apiserver and the kubelet command lines.
+
+### Removal of LocalStorageCapacityIsolation  feature gate
+
+The [Local Ephemeral Storage Capacity Isolation](https://github.com/kubernetes/kubernetes/pull/111513) feature moved to GA in v1.25. It provides support for capacity isolation of local ephemeral storage between pods, such as EmptyDir, so that a pod can be hard limited in its consumption of shared resources by evicting Pods if its consumption of local ephemeral storage exceeds that limit. Hence the feature gate `LocalStorageCapacityIsolation` will be removed in this release.
+
+### Removal of [NetworkPolicyEndPort](https://github.com/kubernetes/kubernetes/pull/110868) feature gate
+
+`endPort` in Network Policy promoted to GA in v1.25. Network Policy providers that support `endPort` field that can be used to specify a range of ports to apply a Network Policy. Previously, each Network Policy could only target a single port. So the feature gate `NetworkPolicyEndPort` will be removed in this release.
+
+Please be aware that `endPort` field must be supported by the Network Policy provider. If your provider does not support endPort, and this field is specified in a Network Policy, the Network Policy will be created covering only the port field (single port)
+
+### Removal of StatefulSetMinReadySeconds feature gate
+
+StatefulSet Controller honoring [minReadySeconds](https://github.com/kubernetes/kubernetes/pull/110896) and mark Pod ready only if Pod is available for the given time mentioned in minReadySeconds. With Promotion of StatefulSet minReadySeconds to GA in v1.25, means `--feature-gates=StatefulSetMinReadySeconds=true` are not needed on kube-apiserver and kube-controller-manager binaries so they'll be removed in this release.
+
+### Removal of [IdentifyPodOS](https://github.com/kubernetes/kubernetes/pull/111229) feature gate
+
+Addition of a new field to the pod spec called os to identify the OS of the containers specified in the pod. From v1.25, the IdentifyPodOS feature is in GA stage and defaults to be enabled hence the `IdentifyPodOS` feature gate will no longer be accepted as a `--feature-gates` parameter in v1.27.
```

Comment:
```suggestion
You can specify the operating system for a Pod, and the feature support for that is stable since
the v1.25 release.
The `IdentifyPodOS` feature gate will be removed for Kubernetes v1.27.
```